content/how-to/bpmn/tune-performance.md

```diff
@@ -53,3 +51,20 @@ A strategy to debug and minimize necessary computation is to break transformatio
 If you use Visual Studio Code, consider the [`jsonata-language`](https://marketplace.visualstudio.com/items?itemName=bigbug.vscode-language-jsonata) extension.
 Similar to a Jupyter notebook, the extension provides an interactive environment to write JSONata expressions and to pass the outputs from one expression into the input of another.
 Besides its benefit for monitoring performance, we have used it to incrementally build complex JSONata in a way that we can  document and share (in the style of literate programming).
+
+## Manage the process context size
+
+{{< notice "note" >}}
+The max size of the process variable context comes from the default max payload size of NATS Jetstreams.
+To increase this size, change your NATS configuration.
```

Comment:
Might be worth linking to the NATS docs here. They have some discussion about reasonable max sizes. The key part is while it'll go up to 64MB (IIRC) they don't recommend going above 8.

Reply:
Definitely a good idea, but are you able to find the link?

My quick search find only these two pages mentioning `maxMsgSize`:

- https://docs.nats.io/nats-concepts/jetstream/streams#retentionpolicy
- https://docs.nats.io/using-nats/developer/develop_jetstream/model_deep_dive

But I couldn't find that specific recommendation.